chapters/dump-reload.rst

```diff
@@ -0,0 +1,127 @@
+Rolling Updates and Query While Indexing in Jina
+================================================
+
+In this section, you will learn how to configure Jina to allow queries while indexing
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Overview
+^^^^^^^^
+
+Querying while indexing refers to the capacity of still being able to
+search your system while data is simultaneously being written to it.
+Within Jina this is possible using the rolling updates feature. This
+means you can have two Flows running at the same time, one for DBMS-type
+operations (adding, updating, deleting data) and one for searching
+(querying). The two systems communicate with each other using a system
+of rolling update calls to the respective Flows.
+
+======================= ==========================================
+**Section**             **Purpose**
+======================= ==========================================
+DBMS and Query Indexers Understand the two Flows you will need
+Dump-Reload             The methods for interacting with the Flows
+Replicas                Configuration needed for uptime
+JinaD                   Remote Flows
+======================= ==========================================
+
+DBMS and Query Indexers
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/dump-reload/dbms-query.jpg
+
+As part of this feature, we provide two types of indexers:
+
+The **DBMS** (Database Management System) indexers store both the
+embedding and the Document’s metadata as one entity (as compared with
+the existing Indexers, which **either**, but **not both**). These
+provide ``index``, ``update``, and ``delete`` methods. They should be
+part of your Indexing Flow, and function as normal Indexers.
```

Comment:
Are we capitalizing Index and Query @FionnD ?

Reply:
Good catch, yes we should because they are in the https://docs.jina.ai/chapters/glossary/ 

**Rule: Anything in the glossary is caplitzed**